src/client/components/user_report_contents.tsx

```diff
@@ -143,111 +98,7 @@ export default function UserReportContents({ index, report, rootDomain }: UserRe
               </tr>
             )}
 
-            {markInvalidMutation.isPending || trackReportActionMutation.isPending ? (
```

Comment:
Moved this code to a separate component since I'm reusing the action buttons in the new view.